src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/snippet/Attribute.java

```diff
@@ -58,8 +58,7 @@
  * This code and its internal interfaces are subject to change or
  * deletion without notice.</b>
  */
-// TODO: uncomment /* sealed */ when minimum boot JDK version >= 17
-public /* sealed */ abstract class Attribute {
+public abstract class Attribute {
```

Comment:
General question:   the commit message is:

> So we can finally use sealed classes where they were originally envisioned.

But, the code edits seem to be backing away from using sealed classes and interfaces.  What happened?

Reply:
This is embarrassing. The commit clearly contradicts its message, and I don't immediately understand why. I'll try to figure out it before integrating. Thanks.

Reply:
I vaguely remember that at some stage I seem to have realized that while `sealed` works for Style, it does not work for Attribute. This is because the attribute is typically queried by the most specific `Class<T extends Attribute>`, which means that the client knows the exact type of the attribute upon return from Attributes.get (note plural attributeS).

Here's another issue that might have contributed to this mess: although the [sealed classes](https://openjdk.java.net/jeps/409) are available, the [exhaustive switch for sealed hierarchies](https://openjdk.java.net/jeps/420) is not.

I will push a commit to make Style `sealed`, which seems reasonable.

Reply:
I vaguely remember that at some stage I seem to have realized that while `sealed` works for Style, it does not work for Attribute. This is because the attribute is typically queried by the most specific `Class<T extends Attribute>`, which means that the client knows the exact type of the attribute upon return from Attributes.get (note plural attributeS).

Here's another issue that might have contributed to this mess: although the [sealed classes](https://openjdk.java.net/jeps/409) are available, the [exhaustive switch for sealed hierarchies](https://openjdk.java.net/jeps/420) is not.

That said, here's what I will do. I will push a commit to make Style `sealed`, which a seems reasonable thing to do. Do you have any objections, Jon? (Asking because you have already reviewed the change.)

Reply:
Thanks for the explanation.   I have no strong preferences either way, with a mild preference for leaving it alone, and improve it later when you may be able to make better use of the declaration.

Reply:
Please clarify: do you want me to revert 93f01b9, push the PR as it is now, or do something else?

Reply:
by défault, push the PR as is, unless you have strong feelings to do otherwise.

Reply:
>> I will push a commit to make Style sealed, which seems a reasonable thing to do. Do you have any objections, Jon?

No objections to that course of action.